### PR TITLE
Vet clinic database:: Queries and update animals table.

### PR DESCRIPTION
**Pull request by joramot**
### **Project requirements**

- Add a column species of type string to your animals table.
- Insert some new data.
- Inside a transaction update the animals table by setting the species column to unspecified.
- Inside a transaction update the animals table by setting the species.
- Inside a transaction delete all records in the animals table, then roll back the transaction.
- Inside a transaction:
      * Delete all animals born after Jan 1st, 2022.
      * Create a savepoint for the transaction.
       * Update all animals' weight to be their weight multiplied by -1.
       * Rollback to the savepoint
        * Update all animals' weights that are negative to be their weight multiplied by -1.
        * 
8 Add a column species of type string to your animals table
9 Inside_a_transaction_update_the_animals_table
10 ROLLBACK_Inside_a_transaction_update_the_animals_table
11 Update the animals table by setting the species column to digimon
12 Update the animals table by setting the species column to pokemon
13 delete all records in the animals table
13 ROLLBACK delete all records in the animals table
14 Savepoint
15 ROLLBACK Savepoint
16 HOW MANY ANIMALS
17 HOW MANY ANIMALS
18 AVG WEIGTH
19 Who escapes the most
20 What is the minimum and maximum weight
21 What is the average number of escape attempts per animal type
Commit transaction

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+
+/* Find all animals not named Gabumon. */
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
+
+/* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+
+ALTER TABLE animals ADD species varchar(100);
```
Comment (MahmoudBakr23) [inline queries.sql]: - [x] Please move this SQL line to the schema where it's required to be as it is a database-changing statement line. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @joramot 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on adding the correct SQL statements as required so far into your app
- Descriptive PR and README

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (joramot) [reply queries.sql]: ok, thank you @MahmoudBakr23

**Review by Deepakdanger — APPROVED** [review]
Hi @joramot, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**